src/lib389/lib389/cli_conf/pwpolicy.py

```diff
@@ -237,6 +237,9 @@ def create_parser(subparsers):
     set_parser.add_argument('--pwdhistory', help="To enable password history set this to \"on\", otherwise \"off\"")
     set_parser.add_argument('--pwdhistorycount', help="The number of passwords to keep in history")
     set_parser.add_argument('--pwdadmin', help="The DN of an entry or a group of account that can bypass password policy constraints")
+    set_parser.add_argument('--pwdadminskipupdates',
+                            help="Set to \"on\" if the Password Admin's password update should not trigger updates to the passowrd state attributes (passwordExpirationtime, passwordHistory, etc).")
```

Comment:
`passowrd` - typo.

But also, you've added the setting to local pwpolicy in CLI, but the UI doesn't have it in local pwpolicy...
So I'm a bit confused if we want it in local pwpolicy or not?

Reply:
ok, the UI had half implemented it in the local policy.  It is all fixed and consistent now